### ISSUE TITLE
Invoking dirvish fails, when not installed from MELPA

### ISSUE THREAD

**Issue by martin5233** [alexluigit/dirvish#90]
I wanted to try out dirvish, but I failed installing it correctly via el-get, which I use for package management. It seems, that dirvish requires at least the dirvish-media extension to be available in the load-path, which is not the case by default. Unfortunately I couldn't find the information, how to activate or deactivate extensions. It would be nice, if you could extend the documentation accordingly.

**Comment by alexluigit** [comment]
Do you have a solution for this? I have no idea how `el-get` works, but I believe all you need is specify the recipe for the package and add "extensions/*.el" to the load-path. Here's how the [recipe](https://github.com/melpa/melpa/blob/master/recipes/dirvish) looks in MELPA.  I'd love to improve the doc after figuring this out. Thanks.

**Comment by martin5233** [comment]
Hi Alex,

thanks for your feedback. I have added the extensions to the load path and this is how the recipe for el-get looks like, which is quite similar to the MELPA recipe:
 ```
              (:name dirvish
                      :type github
                      :pkgname "alexluigit/dirvish"
                      :load-path "extensions")

```
With this addition everything works as expected. You could either simply add this to your git repository or ideally contact the author of el-get to include this recipe in the el-get distribution.

**Comment by alexluigit** [comment]
Thank you for this, I'll put it in README.

**Comment by shipmints** [comment]
Thank you for dirvish.  A nice piece of work.

For regular `use-package` users, I had to do the below to make the nongnu ELPA distribution work.  This looks like the only multi-subdirectory package I have installed in my elpa tree.  Everyone else uses flat trees of lisp files.  It might be worth looking into with the ELPA maintainers, I figured this would be automatic.  Perhaps ELPA needs some special hint in the packaging process.  Or autoloads can be made to automate this at load time when it knows it's own load-file-name.

```elisp
(use-package dirvish
  :init
  (add-to-list 'load-path
               (expand-file-name "extensions"
                                 (file-name-directory (locate-library "dirvish"))))
...
```
